### PR TITLE
'var' parameters are deprecated and will be removed in Swift 3

### PR DESCRIPTION
**Pull request by margem**

### CODE REVIEW

**Comment by matzew** [comment]
@corinnekrych should the Swift 3 bits be developed on a branch, until that Swift version is final ?

File: AeroGearHttp/Http.swift [hunk 1]
```diff
@@ -571,14 +571,15 @@ public class Http {
...
         }
         
         guard let finalURL = NSURL(string: baseURL!) else {return nil}
-        if (url.hasPrefix("/")) {
-            url = url.substringFromIndex(url.startIndex.advancedBy(1))
+        if (partURL.hasPrefix("/")) {
+            partURL = url.substringFromIndex(partURL.startIndex.advancedBy(1))
         }
         
         return finalURL.URLByAppendingPathComponent(url);
```
Comment (corinnekrych) [inline AeroGearHttp/Http.swift]: To make the unit test pass and fix the issue, replace url -> partURL

**Comment by corinnekrych** [comment]
@matzew removing deprecated Swift 2.2 is fine. it's still backward compatible so merging in master is fine.

**Comment by corinnekrych** [comment]
@rafaelmnz Thanks for the contribution, fix the variable issue and we're good to merge!

**Pull request closed without merging**